embassy-stm32/src/rcc/g4.rs

```diff
@@ -41,6 +44,128 @@ pub enum APBPrescaler {
     Div16,
 }
 
+/// PLL clock input source
+#[derive(Clone, Copy, Debug)]
+pub enum PllSrc {
+    HSI16,
+    HSE(Hertz),
+}
+
+impl Into<Pllsrc> for PllSrc {
+    fn into(self) -> Pllsrc {
+        match self {
+            PllSrc::HSE(..) => Pllsrc::HSE,
+            PllSrc::HSI16 => Pllsrc::HSI16,
+        }
+    }
+}
+
+#[derive(Clone, Copy)]
+pub enum PllClkDiv {
```

Comment:
Might want to specify which divider this is for. The P output can be divided between 2 and 31 so this only applies to R and Q dividers.

Reply:
As explained above, renaming to `PllR` would be simpler to understand